### ISSUE TITLE
Bug: ClassCastException

### ISSUE THREAD

**Issue by danielogen** [unlv-evol/PatchIntegrator#1]
**Describe the bug**
Error occurs when trying to collect information on `EXTRACT_AND_MOVE_OPERATION` refactoring operation. This error is likely a result of changes in the `refactoringMiner 2.3` internal code base. See sample error below;

```
Caused by: java.lang.ClassCastException: class gr.uom.java.xmi.UMLInitializer cannot be cast to class gr.uom.java.xmi.UMLOperation (gr.uom.java.xmi.UMLInitializer and gr.uom.java.xmi.UMLOperation are in unnamed module of loader 'app')
```

**To Reproduce**
1. `git clone` the project locally and setup in intellij 
2. Run the project to reproduce the error

**Expected behavior**
Success extract information on `EXTRACT_AND_MOVE_OPERATION` refactoring operation

**Screenshots**
<img width="1383" alt="Screen Shot 2023-04-11 at 12 29 50 PM" src="https://user-images.githubusercontent.com/15224992/231117752-e8570aaf-a852-42d0-9e52-dc45752377f0.png">

**Desktop (please complete the following information):**
 - OS: MacOs Big Sur
 - Version: 11.1

**Additional context**
- None

**Comment by johnxu21** [comment]
If this issue is resulting from refactoring miner, I suggest you post it on
refactoringminer's repo.

**Comment by danielogen** [comment]
This is okay... I also see that `RefactoringMiner` released some updates (2.3.0 -> 2.4.0) about a month ago. I will try it out to see if it fixes the issue.

see: [https://github.com/tsantalis/RefactoringMiner/releases/tag/2.4.0](https://github.com/tsantalis/RefactoringMiner/releases/tag/2.4.0)

**Comment by danielogen** [comment]
This is issues is not resolved by RefractoringMiner 3.0.9 and a small change in the project code
